lua/orgmode/org/indent.lua

```diff
@@ -194,8 +195,15 @@ local function foldtext()
   return line .. config.org_ellipsis
 end
 
+local function setup()
+  if config.org_indent_mode == 'virtual_indent' then
```

Comment:
We need to check if value is `noindent` here.

Reply:
Why? And if the user doesn't want any indentation? I see three options for using indentation:
1. No indentation
2. Virtual indentation
3. Real indentation

Reply:
No indentation was initially meant to be used as virtual indentation.
If you check in the Emacs docs, there are two values, `indent` and `noindent` (https://orgmode.org/manual/Org-Indent-Mode.html)

Problem is that I set the `indent` as default, where it should actually be noindent. `indent` option is actually a `virtual indent`. I'll fix that and announce it on the breaking change issue so people are aware. Once I do that you will actually have to check for `indent` value.

Reply:
Why not have 3 options? Yes, this is somewhat at odds with emacs, but it makes the configuration more flexible.

Reply:
`noindent` option as it is now is not useful at all. Everything is aligned to the left. As far as I know, people are now using it only to get proper formatting with other tools, like Orgzly. What we have now by default will become `noindent`, and this PR will introduce a proper `indent` option. There's no need for 3rd.

Reply:
I know it may be weird, but some people don't use indentation. Some time ago I came across [this discussion](https://www.reddit.com/r/emacs/comments/97naje/what_is_everyones_org_mode_indentation_preferences) and found out that not everyone needs indents (see the first comment).

In addition, if you break backward compatibility with indent, quite a lot of org files can be visually broken. I'm not sure if this is a good idea.

Reply:
We can allow that functionality by doing the same thing that Emacs does, via `org-adapt-indentation` option. The discussion has a few comments explaining how it works.